aten/src/ATen/native/native_functions.yaml

```diff
@@ -1414,6 +1414,16 @@
 
 - func: exp.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
 
+- func: exp2(Tensor self) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: exp2_(Tensor(a!) self) -> Tensor(a!)
+  use_c10_dispatcher: full
+  variants: function, method
```

Comment:
It's interesting that we're inconsistent in native_functions.yaml about whether the inplace variants of ops have or don't have function variants. Being consistent with exp here sounds good. PyTorch should probably review this inconsistency in the future.